src/plugins/github/README.md

```diff
@@ -0,0 +1,175 @@
+# GitHub Plugin
+
+This is the SourceCred GitHub plugin. It creates a cred graph based on data
+from [GitHub].
+
+[GitHub]: https://github.com/
+
+## Nodes
+
+The GitHub plugin creates the following kinds of nodes:
+
+### Repo
+
+A GitHub repository, e.g. [sourcecred/sourcecred]. It will be directly
+connected to all of the pulls and issues in the repository. Setting a weight on
+the repository will have no effect (i.e. they do not mint cred). This will
+change when we switch to [CredRank].
+
+[CredRank]: https://github.com/sourcecred/sourcecred/issues/1686
+
+[sourcecred/sourcecred]: https://github.com/sourcecred/sourcecred
+
+### Issue
+
+A GitHub issue, e.g. [sourcecred/sourcecred#40]. Issues are connected to their
+author(s), to entities that they reference, to their comments, and to the
+containing GitHub repository. If issues are given a positive weight, then each
+issue will mint cred. Note that this could be abused by spammers.
+
+[sourcecred/sourcecred#40]: https://github.com/sourcecred/sourcecred/issues/40
+
+### Pull Request
+
+A GitHub pull request, e.g. [sourcecred/sourcecred#35][pull]. Pulls are
+connected to their author(s), to entities they reference, to their comments,
+their reviews, to their containing repository, and, (if merged) to the commit
+that they merged as. If pulls are given a positive weight, then each pull will
+mint cred, regardless of whether or not it merged. Note that this could be
+abused by spammers. There's a proposal ([merge-minted cred]) to have pulls mint
+cred only when they merge.
```

Comment:
Question, mentioning proposals has a limited lifespan. I'm wondering if you'd deliberately chosen some strategy here. Do we generally want to warn for tentitive anti-patterns even if the information might go stale quickly? And are these documents well suited?

Reply:
I agree about content (links) going stale. Even if the content is written directly here, it still has an expiration in the sense that it might go out of date. But is it more likely that it would be seen if written directly in the README vs. a link?

Reply:
I removed the link, and moved mention of the proposal (in general terms) to the "status and caveats" section.